### ISSUE TITLE
use aria-hidden on the icons of the IAP home page

### ISSUE THREAD

**Issue by Elisa0122** [GCTC-NTGC/gc-digital-talent#6249]
## ❌ Current behaviour (issue)
When voice-over is used, it read the image without the description.

**WCAG 2.1 Criteria:** 1.1.1 non-text Content: All [non-text content](https://www.w3.org/TR/WCAG22/#dfn-non-text-content) that is presented to the user has a [text alternative](https://www.w3.org/TR/WCAG22/#dfn-text-alternative) that serves the equivalent purpose.

### How to reproduce

1. Go to https://uat-talentcloud.tbs-sct.gc.ca/en/indigenous-it-apprentice
2. use apple Voice Over on chrome
3. Start reading by using VO + A

## ✔ Expected behaviour (fix)

1. The image must not be read because it is [pure decoration] it is used only for visual formatting, and the text below already describes the purpose of the icon.
<img width="1481" alt="Screenshot 2023-04-11 at 3 41 55 PM" src="https://user-images.githubusercontent.com/83783101/231271365-759a9913-634a-4086-8d52-25300b1ff24d.png">

### Acceptance Criteria

- [ ] Apple Voice Over screen reader cannot programmatically determine that there is an image.
- [ ] aria-hidden used to hide the four icons

## Additional information (optional)

1. How does it impact users?
An image tag with no alt text can be a significant barrier for users of screen readers. Without alt text, the screen reader may not convey the image's purpose or meaning, frustrating users who may miss important information conveyed through the image.

2. Related issues, suggestions how to fix
Use aria-hidden="true" on the SVG icons

**Comment by petertgiles** [comment]
Thanks for this bug, Elisa.  Just to confirm, the action that should be taken is to apply `aria-hidden="true"` to the four icons?

**Comment by Elisa0122** [comment]
Hi, yes I am assuming by using aria-hidden="true" the screen reader will just skip the icon and read the text right. Am I right?

**Comment by petertgiles** [comment]
@esizer Do you think aria-hidden=true is the way to go here?  I also saw [this](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden):
> aria-hidden="true" will remove the entire element from the accessibility API.
role="presentation" and role="none" will remove the semantic meaning of an element while still exposing it and its content to assistive technology.

**Comment by esizer** [comment]
> @esizer Do you think aria-hidden=true is the way to go here? I also saw [this](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden):
> 
> > aria-hidden="true" will remove the entire element from the accessibility API.
> > role="presentation" and role="none" will remove the semantic meaning of an element while still exposing it and its content to assistive technology.

For these icons, `aria-hidden` seems appropriate since they appear to be purely decorative. Unless someone wants to make the case that they are semantic. I don't think anyone would lose any sort of value from these being removed entirely from the tree.

**Comment by RM-1978** [comment]
Elisa and I agree @esizer, aria-hidden seems appropriate for these decorative icons. We discussed the semantic value/role of these specific icons in this instance and determined that the text below presents enough key information that the icon descriptiond would not necessarily add meaning to this page that is not already  covered.
